src/EventHub/EventHub/help/Az.EventHub.md

```diff
@@ -89,14 +87,14 @@ Regenerates an EventHub SAS key
 ### [New-AzEventHubKeyVaultPropertiesObject](New-AzEventHubKeyVaultPropertiesObject.md)
 Create an in-memory object for KeyVaultProperties.
 
-## [New-AzEventHubLocationsNameObject](New-AzEventHubLocationsNameObject.md)
-Construct an in-memory NamespaceReplicaLocation object
+### [New-AzEventHubLocationsNameObject](New-AzEventHubLocationsNameObject.md)
+Create an in-memory object for NamespaceReplicaLocation.
 
 ### [New-AzEventHubNamespace](New-AzEventHubNamespace.md)
 Creates an EventHub Namespace
 
 ### [New-AzEventHubSchemaGroup](New-AzEventHubSchemaGroup.md)
-Create an EventHub schema group.
+create an EventHub schema group.
```

Comment:
The first letter c is not capitalized.

Reply:
Update to this PR forthcoming, but incorrect in [the Synopsis of the reference doc](https://github.com/Azure/azure-powershell/blob/main/src/EventHub/EventHub/help/New-AzEventHubSchemaGroup.md#synopsis).